src/services/reCaptcha.js

```diff
@@ -0,0 +1,24 @@
+const axios = require("axios");
+
+class ReCaptcha {
+  constructor(userToken) {
+    this.userToken = userToken;
+    this.url = "https://www.google.com/recaptcha/api/siteverify";
+    this.secret = process.env.RECAPTCHA_SECRET;
+  }
+
+  async validateUser() {
+    const googleResponse = await this.askGoogle();
+    return googleResponse.data.success;
+  }
+
+  async askGoogle() {
+    const postUrl = `${this.url}?secret=${this.secret}&response=${this.userToken}`;
+
+    const response = await axios.post(postUrl, {});
```

Comment:
What happens on a network error? I guess this will throw an error and we will return a 500. That seems OK for now. Maybe add a comment to consider a retry in the future.

Reply:
Yeah, I'll add a comment!